course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -0,0 +1,101 @@
+### The `if` statement
+- the `if` statement is used to check if a conditional is `True`
```

Comment:
You could use `if` to check if the conditional is `False`, `==`, `!=` and the rest of the logical conditions too. How about something like:

> The if statement is used for conditional execution.

Reply:
My thought here is that if we can check that something is True, we are automatically checking if it's False, since those two are complementary terms. 
By conditional, I mean (that's the way I understood it as well) a statement that contains any conditional logic inside of it, which is independent of symbols that we use as long as the logic is consistent and able to execute properly.
A help from @jslvtr could maybe come in handy here, just so we can check that we are all using proper terms

Reply:
Yep :+1: looks good, and the change reads well!